logs/logger.js

```diff
@@ -0,0 +1,30 @@
+// logger.js
+const winston = require("winston");
+
+// Define the log levels and colors (optional)
+const logLevels = {
+  error: "red",
+  warn: "yellow",
+  info: "green",
+  debug: "blue",
+};
+
+winston.addColors(logLevels);
+
+const logger = winston.createLogger({
+  level: "debug",
```

Comment:
We can remove this. By having this here, it means we can _only_ use `logger` to log DEBUG level logs. So `logger.info` and `logger.error` won't print anything, only `logger.debug` will. We _could_ do it that way and create separate loggers for the info and error levels but that feels too repetitive for this case.

Rather, we can leave out the `level` in this `createLogger` function; and then that will allow us to choose what level we want to use each time when we log things.